client/my-sites/importer/style.scss

```diff
@@ -38,6 +38,10 @@
 	&.wordpress {
 		background-color: $blue-wordpress;
 	}
+
+	&.medium {
+		background-color: darken( $gray-light, 31% );
```

Comment:
is there a particular reason 31% was chosen? I have no problem with this code, but it stands out :smile: